src/test/java/edu/ucsb/cs156/gauchoride/controllers/RideControllerTests.java

```diff
@@ -153,8 +153,10 @@ public void test_that_logged_in_user_can_get_by_id_when_the_id_exists_and_user_i
                                 .startTime("2:00PM")
                                 .endTime("3:15PM")
                                 .dropoffLocation("South Hall")
```

Comment:
On the next three lines, you have comments such as `// MODIFIED`, and `// ADDED`.

Most professional developers would consider these to be clutter that should not be in the code base.  Git and Github track this for us, so it's not necessary, and make the code less readable.  Please consider deleting these, and not adding these in the future.

Reply:
@pconrad when you do, please include the long-term relevancy of comments on how those that explain “why” tend to stay relevant longer than those that explain “what”